frontend/src/components/chat/ChatInterface.tsx

```diff
@@ -110,11 +112,30 @@ function ChatInterface() {
     }
   }, [curAgentState, dispatch, messages.length, t]);
 
+  const toggleMute = () => {
+    const cookieName = "audio";
+    setIsMuted(!isMuted);
+    if (!isMuted) {
+      document.cookie = `${cookieName}=;`;
+      console.log("Audio muted");
+    } else {
+      document.cookie = `${cookieName}=on;`;
+      console.log("Audio unmuted");
+      beep();
+    }
+  };
   return (
     <div className="flex flex-col h-full bg-neutral-800">
       <div className="flex items-center gap-2 border-b border-neutral-600 text-sm px-4 py-2">
         <IoMdChatbubbles />
         Chat
+        <div className="ml-auto">
```

Comment:
This is a nice button, but one suggestion: could we please put this next to the settings button on the bottom-right? Putting it on the top makes it a bit difficult to parse the various locations for settings buttons, so it'd be good to put them in the same place.

Reply:
https://github.com/OpenDevin/OpenDevin/pull/2435 The autonomous button will be there too.  Aren't these frequent settings?

Reply:
To clarify, I don't mean to put them in global settings, but rather to put the button *next to* the settings button on the main screen.
It's a lot of cognitive load to have many different places where you can click buttons on the screen.

Reply:
Moved the button to the left of the settings button.